### PR TITLE
Added additional features to pipeline, updated CRD

### PR DESCRIPTION
**Pull request by nathan-weinberg**
- Exposed `eBPF` memory limit and `router-perf` workload options for configuration
- Added `DESTROY_CLUSTER` option to cleanup stage
- Added `PROFILE_PORT` to flowcollector CRD
- Minor updates to upstream Dittybopper config

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -596,7 +633,9 @@ pipeline {
                             booleanParam(name: 'CERBERUS_CHECK', value: params.CERBERUS_CHECK),
                             booleanParam(name: 'MUST_GATHER', value: true),
                             string(name: 'JENKINS_AGENT_LABEL', value: params.JENKINS_AGENT_LABEL),
-                            booleanParam(name: 'GEN_CSV', value: false)
+                            booleanParam(name: 'GEN_CSV', value: false),
+                            string(name: 'LARGE_SCALE_CLIENTS', value: params.LARGE_SCALE_CLIENTS),
```
Comment (Amoghrd) [inline Jenkinsfile:637]: where is this var used after setting?

Comment (nathan-weinberg) [reply Jenkinsfile:637]: @Amoghrd in the router-perf job...

**Comment by nathan-weinberg** [comment]
Merged by accident by pushing to `upstream` instead of `origin` - fixed this and will reopen a PR

**Pull request merged**